app/views/claims/bank_details.html.erb

```diff
@@ -4,6 +4,7 @@
       <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
       <%= form_for current_claim, url: claim_path do |form| %>
+        <%= form_group_tag current_claim do %>
```

Comment:
Because of this outer group both fields bank fields end up with the red line, even when only one of the fields have an error. That may be OK but thought I'd point it out.

Reply:
Yeah, it's a good point. I think I need to revisting the GOVUK design system…